project/Dependencies.scala

```diff
@@ -113,4 +113,11 @@ object Dependencies {
 
   val scalaXml = scala211Module("scala-xml", "1.0.6")
   val scalaParsers = scala211Module("scala-parser-combinators", "1.0.5")
+
+  def log4jVersion = "2.8.1"
+  val log4jApi = "org.apache.logging.log4j" % "log4j-api" % log4jVersion
+  val log4jCore = "org.apache.logging.log4j" % "log4j-core" % log4jVersion
+  val log4jSlf4jImpl = "org.apache.logging.log4j" % "log4j-slf4j-impl" % log4jVersion
+  // specify all of log4j modules to prevent misalignment
+  val log4jDependencies = Vector(log4jApi, log4jCore, log4jSlf4jImpl)
```

Comment:
Hmm, I guess it does make sense that sbt is an application that defines which slf4j backend to use. This will be part of the migration process for a few plugins. IMO we should highlight it somewhat in the release notes.